monkestation/code/game/machinery/doors/door.dm

```diff
@@ -0,0 +1,4 @@
+/obj/machinery/door/crush()
```

Comment:
Don't double define the proc.
Go put this in the actual /obj/machinery/door/proc/crush() definition and comment around it

Reply:
Modularity sucks.